src/components/editor/MermaidGeneratorResultPreview.tsx

```
@@ -0,0 +1,40 @@
+      </div>
+
+      <div className="space-y-2">
+        <Label>プレビュー</Label>
+        {previewError ? (
+          <div className="rounded-md bg-destructive/10 p-4 text-sm text-destructive">
+            {previewError}
+          </div>
+        ) : previewSvg ? (
+          <div
+            className="flex justify-center overflow-auto rounded-md border bg-white p-4 dark:bg-gray-900"
+            dangerouslySetInnerHTML={{ __html: previewSvg }}
```

Biome (2.4.6)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)